ubuntu/gpu/cuda-11.0/base/Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM nvidia/cuda:11.0.3-cudnn8-runtime-ubuntu18.04
 
 # Disable NVIDIA repos to prevent accidental upgrades.
 RUN cd /etc/apt/sources.list.d && \
-    mv cuda.list cuda.list.disabled && \
-    mv nvidia-ml.list nvidia-ml.list.disabled
+    mv cuda.list cuda.list.disabled
```

Comment:
Is the ml.list not relevant anymore?

Reply:
yes, I did `ls /etc/apt/sources.list.d ` in the nvidia/cuda:11.0.3-cudnn8-runtime-ubuntu18.04 image, they only contain `cuda.list ` now. seemed like nvidia updated the it image